### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by jkroepke**

### CODE REVIEW

File: sts-e2e-tests/src/test/java/de/adorsys/sts/tests/e2e/tokenexchange/TokenExchangeControllerJpaTest.java [hunk 1]
```diff
@@ -55,6 +55,6 @@ void tokenExchangeTest() {
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.issued_token_type").value("urn:ietf:params:oauth:token-type:access_token"))
                 .andExpect(jsonPath("$.token_type").value("Bearer"))
-                .andExpect(jsonPath("$.expires_in").value("84580358"));
+                .andExpect(jsonPath("$.expires_in").value("315576000"));
```
Comment (valb3r) [inline sts-e2e-tests/src/test/java/de/adorsys/sts/tests/e2e/tokenexchange/TokenExchangeControllerJpaTest.java:58]: This change is not necessary when injecting Clock (no changes needed in the test)

File: sts-e2e-tests/src/test/resources/fixture/test_token.txt [hunk 2]
```diff
@@ -1 +1 @@
-eyJraWQiOiJhYmMiLCJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwczovL3lvdXItaWRwLWhvc3RuYW1lL2F1dGgvcmVhbG1zL3lvdXItcmVhbG0iLCJodHRwOi8vZXhhbXBsZS5jb20vaXNfcm9vdCI6dHJ1ZSwic3ViIjoiZGV2IiwibmFtZSI6ImUyZSB0ZXN0IiwiaWF0IjoxNTE2MjM5MDIyLCJleHAiOjE2MDA4MTkzODB9._S742nPRhcog8sDXPgy94pwVYcqgHiGEPn-jn2YEQbY
\ No newline at end of file
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImFiYyJ9.eyJpc3MiOiJodHRwczovL3lvdXItaWRwLWhvc3RuYW1lL2F1dGgvcmVhbG1zL3lvdXItcmVhbG0iLCJodHRwOi8vZXhhbXBsZS5jb20vaXNfcm9vdCI6dHJ1ZSwic3ViIjoiZGV2IiwibmFtZSI6ImUyZSB0ZXN0IiwiaWF0IjoxNTE2MjM5MDIyLCJleHAiOjE4MzE4MTUwMjJ9.uleDOpiFYWoGQzeKh6TTFgKzSIKpZtlT820A46f7T2k
```
Comment (valb3r) [inline sts-e2e-tests/src/test/resources/fixture/test_token.txt:1]: This change is not necessary when injecting Clock (no changes needed in the test)

**Comment by jkroepke** [comment]
@valb3r Should I remove the new token or could it leave it as it?

**Comment by valb3r** [comment]
> @valb3r Since it will be work with both token (old and new one) should I remove the new token or could I leave it as it?

@jkroepke I suggest keeping old token as it works

File: sts-e2e-tests/src/test/java/de/adorsys/sts/tests/e2e/tokenexchange/TokenExchangeControllerJpaTest.java (hunk 1 above)
Comment (jkroepke) [reply sts-e2e-tests/src/test/java/de/adorsys/sts/tests/e2e/tokenexchange/TokenExchangeControllerJpaTest.java:58]:
```suggestion
                .andExpect(jsonPath("$.expires_in").value("84580358"));
```

File: sts-e2e-tests/src/test/resources/fixture/test_token.txt (hunk 2 above)
Comment (jkroepke) [reply sts-e2e-tests/src/test/resources/fixture/test_token.txt:1]:
```suggestion
eyJraWQiOiJhYmMiLCJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwczovL3lvdXItaWRwLWhvc3RuYW1lL2F1dGgvcmVhbG1zL3lvdXItcmVhbG0iLCJodHRwOi8vZXhhbXBsZS5jb20vaXNfcm9vdCI6dHJ1ZSwic3ViIjoiZGV2IiwibmFtZSI6ImUyZSB0ZXN0IiwiaWF0IjoxNTE2MjM5MDIyLCJleHAiOjE2MDA4MTkzODB9._S742nPRhcog8sDXPgy94pwVYcqgHiGEPn-jn2YEQbY
```

**Review by valb3r — APPROVED** [review]

**Pull request merged**